services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -136,7 +136,7 @@ def create_app():
 
     # MODULES SETUP --------------
 
-    app.state.shared_store = SharedStore()
+    setup_shared_store(app)
```

Comment:
what is a store?
is that the attached docker volume? in that case can we have an explicit name such as
```setup_docker_volume``` or at least something like setup_shared_storage or setup_shared_folders?

Reply:
The `shared_store` is an area of memory dedicated to storing some information usually the docker_compose spec and the started container names.

Reply:
so it's a kind of app runtime storage... maybe we can find something better...

Reply:
IMO this is the definition of ``app.state`` provided by FastAPI.

Reply:
actually @pcrespov, I think you are right ;)